h2o-core/src/main/java/water/util/MRUtils.java

```diff
@@ -128,6 +130,57 @@ public void map(Chunk[] cs, NewChunk[] ncs) {
     @Override public void reduce( ClassDist that ) { ArrayUtils.add(_ys,that._ys); }
   }
 
+  /**
+   * Compute the class distribution for qusibinomial distribution from a class label vector
+   * (not counting missing values)
+   */
+  public static class ClassDistQuasibinomial extends MRTask<ClassDistQuasibinomial> {
+    final int _nclass;
+    protected double[] _ys;
+    protected Double[] _domain;
+    
+    public ClassDistQuasibinomial(String[] domain) { 
+      _nclass = 2;
+      _domain = new Double[]{Double.valueOf(domain[0]), Double.valueOf(domain[1])};
+      
+    }
+    
+    public final double[] dist() {
+      return _ys; 
+    }
+    
+    public final double[] relDist() {
+      final double sum = ArrayUtils.sum(_ys);
```

Comment:
with 0 weights and NA responses we can have sum = 0, can't we?

Reply:
I think the sum cannot be zero. At least one domain has to be set; there is a check for this before model start building.

Reply:
even with 0 weights?